content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -185,6 +185,11 @@ These instructions are for Kubernetes {{< skew currentVersion >}}.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
+{{< note >}}
+In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
+you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
+{{< /note >}}
```

Comment:
Even better IMO (but I didn't test it):
```suggestion
   {{< note >}}
   In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
   you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
   {{< /note >}}
```

That makes the note as logically part the existing ordered list item.

Reply:
It also makes sense to me, but for soms reason (maybe a bug on this tabbed design) it seems to break. The way i commited works despite not perfectly aligned.